sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -127,7 +127,9 @@ flows AS (
       AND user_id IS NOT NULL
     ) AS pay_setup_engage_with_uid,
     -- new fxa after entering the email
-    LOGICAL_OR(event_type IN ("fxa_pay_setup - 3ds_complete", "fxa_pay_setup - success")) AS pay_setup_complete,
+    LOGICAL_OR(
+      event_type IN ("fxa_pay_setup - 3ds_complete", "fxa_pay_setup - success")
+    ) AS pay_setup_complete,
```

Comment:
is this change needed?

Reply:
Not required for the change. But it's a re-formatting of the file, autogenerated when running the files formatting check.